packages/block-library/src/term-description/edit.js

```diff
@@ -13,18 +13,27 @@ import {
 	AlignmentControl,
 } from '@wordpress/block-editor';
 
+/**
+ * Internal dependencies
+ */
+import { useTermDescription } from './use-term-description';
+
 export default function TermDescriptionEdit( {
 	attributes,
 	setAttributes,
 	mergedStyle,
+	context: { termId, taxonomy },
```

Comment:
I'm in the process of getting some more context for terms query block and saw this PR. Where this context comes from?  Have we updated something else that auto injects them like `postId and postType`?

I tried in the editor and even site editor in category specific template and these are always undefined.

Reply:
This context needs to be provided by some parent block. To my knowledge, there is currently no block that does that in Core (though maybe the Term Query will?). This PR adds the possibility to receive that context. In WooCommerce, the Featured Category block provides context to inner blocks, for example.

Reply:
It pulls through the term description for me in the editor, although it looks like it doesn't work on the server side yet, which we can fix soon:

<img width="686" height="677" alt="image" src="https://github.com/user-attachments/assets/bbe2e2eb-9740-49a8-a468-c76221f6242c" />